Description: The subprocess call results in OSError "No such file or directory"
             since git is not available when sphinx-build is called during
             package builds.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: not-needed

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -158,11 +158,12 @@
 # If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
 # using the given strftime format.
 #html_last_updated_fmt = '%b %d, %Y'
-git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local -",
-   "n1"]
-html_last_updated_fmt = subprocess.Popen(git_cmd,
-                                         stdout=subprocess.PIPE).\
-                                         communicate()[0]
+html_last_updated_fmt = ''
+#git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local -",
+#   "n1"]
+#html_last_updated_fmt = subprocess.Popen(git_cmd,
+#                                         stdout=subprocess.PIPE).\
+#                                         communicate()[0]
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
